requirements.txt

```diff
@@ -1,9 +1,11 @@
-av==10.*
+av==11.*
 ctranslate2>=4.0,<5
 huggingface_hub>=0.13
 tokenizers>=0.13,<0.16
 onnxruntime>=1.14,<2 
 transformers #4.36.0
+pyannote-audio>=3.1.1
+pandas>=2.1.4
 #torch>=2.1.1+cu121
```

Comment:
Why are these commented out? Should they be deleted instead?

Reply:
Ah these are needed. Updated the requirements file. Please have a look